tests/core/test_log.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import pytest
+
+from pytest_ethereum.testing import Log
+
+
+logging.getLogger("evm").setLevel(logging.INFO)
+
+
+@pytest.fixture
+def ping(vyper_project_dir, vy_deployer):
+    return vy_deployer.deploy("ping")
+
+
+def test_log_is_present(ping, w3):
+    ping_instance = ping.deployments.get_contract_instance("ping")
+    tx_hash = ping_instance.functions.ping(b"1", b"2").transact()
+    receipt = w3.eth.waitForTransactionReceipt(tx_hash)
+    assert Log(ping_instance.events.Ping, b"1", b"2").is_present(receipt)
+    assert Log(ping_instance.events.Ping, b"1").is_present(receipt)
+
+
+def test_log_match(ping, w3):
+    ping_instance = ping.deployments.get_contract_instance("ping")
+    tx_hash = ping_instance.functions.ping(b"1", b"2").transact()
+    receipt = w3.eth.waitForTransactionReceipt(tx_hash)
+    assert Log(ping_instance.events.Ping, _first=b"1", _second=b"2").match(receipt)
+    assert Log(ping_instance.events.Ping, _first=b"1").match(receipt) is False
+    assert Log(ping_instance.events.Ping, _third=b"1").match(receipt) is False
+    assert Log(ping_instance.events.Ping, b"1").match(receipt) is False
```

Comment:
Add a comment that "under-specified match doesn't pass"

(or is it that no kwarg won't pass? Can't tell! No comment lol)